com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs

```diff
@@ -5308,7 +5293,7 @@ void SendGeometryGraphicsBuffers(CommandBuffer cmd, HDCamera hdCamera)
             VFXCameraBufferTypes neededVFXBuffers = VFXManager.IsCameraBufferNeeded(hdCamera.camera);
             needNormalBuffer |= ((neededVFXBuffers & VFXCameraBufferTypes.Normal) != 0 || (externalAccess & HDAdditionalCameraData.BufferAccessType.Normal) != 0);
             needDepthBuffer |= ((neededVFXBuffers & VFXCameraBufferTypes.Depth) != 0 || (externalAccess & HDAdditionalCameraData.BufferAccessType.Depth) != 0);
-            if (hdCamera.frameSettings.IsEnabled(FrameSettingsField.RayTracing) && GetRayTracingState() || ValidIndirectDiffuseState(hdCamera))
+            if (hdCamera.frameSettings.IsEnabled(FrameSettingsField.RayTracing) && GetRayTracingState() || GetIndirectDiffuseMode(hdCamera) != IndirectDiffuseMode.Off)
```

Comment:
Anis: We use normal buffer from previous frame, but do we need the depth buffer for SSGI?
Also are those buffer require for RTGI?

Reply:
the depth buffer from the previous frame is used for SSGI and both are required for RTGI